components/helm-broker/internal/ybundle/populator_test.go

```diff
@@ -38,7 +39,6 @@ func TestPopulatorInitHappyPath(t *testing.T) {
 	assert.NoError(t, err)
 
 	logSink.AssertLogged(t, logrus.InfoLevel, "Bundle with name [meme] and version [0.10.0] successfully stored")
-
 }
 
 func TestPopulatorErrorOnGetIndexFile(t *testing.T) {
```

Comment:
`TestPopulatorErrorOnGetIndexFile` but we expect `require.NoError(t, err)`
same for others

Reply:
Test method names changed